### PR TITLE
Add timestamp tooltip

### PR DESCRIPTION
**Pull request by Pytal**
Closes nextcloud/photos#598

Related PR: nextcloud/server#27051

### CODE REVIEW

File: src/components/AppSidebar/AppSidebar.vue [hunk 1]
```diff
@@ -204,7 +204,10 @@
 								</form>
 							</template>
 							<!-- secondary title -->
-							<p v-if="subtitle.trim() !== ''" class="app-sidebar-header__subtitle">
+							<p
+								v-if="subtitle.trim() !== ''"
+								v-tooltip.top-start="subtitleTooltip"
```
Comment (skjnldsv) [inline src/components/AppSidebar/AppSidebar.vue:209]:
```suggestion
								v-tooltip.auto="subtitleTooltip"
```

Maybe it's best to leave this automatically positioned?

Comment (raimund-schluessler) [reply src/components/AppSidebar/AppSidebar.vue:209]: Yes, `top-start` looks a bit off. `auto` should be better.

**Review by ma12-co — APPROVED** [review]

**Comment by raimund-schluessler** [comment]
@Pytal Could you please squash your commit and add your signoff? Then it's good to go.

**Review by juliushaertl — APPROVED** [review]

**Review by raimund-schluessler — APPROVED** [review]

**Pull request merged**